src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -113,6 +113,39 @@ class ConvFusingTest : public BaseFusingTest<convolution_test_params> {
     }
 };
 
+class ConvReorderFusingTest : public BaseFusingTest<convolution_test_params> {
+public:
+    void execute(convolution_test_params& p) {
+        auto input_prim = get_mem(get_input_layout(p));
+        network network_not_fused(this->engine, this->topology_non_fused, bo_not_fused);
+        network network_fused(this->engine, this->topology_fused, bo_fused);
+        network_fused.set_input_data("input", input_prim);
+        network_not_fused.set_input_data("input", input_prim);
+
+        compare(network_not_fused, network_fused, p, true);
+        auto find_conv = [](primitive_info& p) -> bool {
+            if (p.original_id == "conv_prim")
```

Comment:
in your tests below, no "conv_prim" primitive_id is used. Is this needed?

Reply:
fixed (conv_out to conv_prim)